kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -284,6 +284,7 @@ define([
                             channelObject = { [jcm.CHANNELS.CELL]: channel };
                         }
 
+                        // listen for job-related bus messages
```

Comment:
I found all the runtime bus listeners and added this comment so it's easier to pull them out in a search